pkg/message/slack.go

```diff
@@ -44,6 +48,40 @@ func (s *Slack) SendPullRequestEvent(URL, title, author string) error {
 	return s.sendMessage(asJSON)
 }
 
+func (s *Slack) NotifyMergeRequestCreated(mergeRequest models.MergeRequestInfo) error {
+	for _, channel := range s.ChannelsForMergeRequest(mergeRequest) {
+		var m = message{
+			Text: fmt.Sprintf("%s <%s|%s> by %s", s.message, mergeRequest.ObjectAttributes.URL, mergeRequest.ObjectAttributes.Title, mergeRequest.User.Name),
+			IconURL: s.avatar,
+			Username: s.username,
+			Channel: channel,
+		}
+
+		asJSON, err := json.Marshal(m)
+		if err != nil {
+			return err
+		}
+
+		return s.sendMessage(asJSON)
```

Comment:
It will return after sending the first message, better do:
```
err = s.sendMessage(asJSON)
if err != nil {
    return err
}
```

Reply:
Good catch! I'll add a test for it... 👍🏻